content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -22,7 +22,7 @@ for your specific needs. The following built-in configuration profiles are curre
     This profile is recommended for production deployments and for
     {{< gloss "primary cluster" >}}primary clusters{{< /gloss >}} in a
     [multicluster mesh](/docs/ops/deployment/deployment-models/#multiple-clusters).
-    You can display the default settings by running the `istioctl profile dump` command.
+    You can view the default profile setting [here](https://github.com/istio/istio/blob/master/manifests/profiles/default.yaml) .
```

Comment:
```suggestion
```

Please link "Istio’s built-in configuration profiles" above to the profiles directory.  You should have to use `{{< github_file >}}/manifests/profiles` as the target (it will update with the correct version for whichever docs you are looking at.)

Reply:
let me push a fix for this.